Example/Source/Examples/Demos/Twitter/MockTwitterViewController.swift

```diff
@@ -226,8 +226,13 @@ extension MockTwitterViewController: WhoToFollowBrickDataSource {
 }
 
 extension MockTwitterViewController: CollectionBrickCellDataSource {
+    
+    func registerBricks(for cell: CollectionBrickCell) {
+        cell.brickCollectionView.registerBrickClass(DailySalesBrick.self)
```

Comment:
As you are registering just the class, shouldn't you add these brickTypes in the initializer of the CollectionBrick?